e2e/commands/start/start_test.go

```diff
@@ -311,4 +312,20 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 			Expect(session.Out.Contents()).To(ContainSubstring("YOUR ANALYSIS HAD FINISHED WITHOUT ANY VULNERABILITY!"))
 		})
 	})
+
+	When("--custom-path-rules is passed", func() {
+		customRulesJson := filepath.Join(os.TempDir(), fmt.Sprintf("horusec-custom-rules.json"))
+
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath:     projectPath,
+				testutil.StartFlagCustomRulesPath: customRulesJson,
+			}
+		})
+
+		It("Checks if the custom rules path property was set", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`\"custom_rules_path\": \"%s\"`, customRulesJson)))
```

Comment:
Use the `testutil.NormalizePathToAssertInJSON()` function to passed test on Windows.

`Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf('\"custom_rules_path\": \"%s\"', testutil.NormalizePathToAssertInJSON(customRulesJson))))`

Reply:
adjusted